undertow/src/main/java/org/wildfly/extension/undertow/UndertowTransformers.java

```diff
@@ -118,13 +120,22 @@ private static void registerTransformersWildFly18(ResourceTransformationDescript
     }
 
     private static void registerTransformersWildFly16(ResourceTransformationDescriptionBuilder subsystemBuilder) {
+        subsystemBuilder.getAttributeBuilder()
+                .addRejectCheck(RejectAttributeChecker.DEFINED, UUID_INSTANCE_ID)
+                .setDiscard(DiscardAttributeChecker.ALWAYS, UUID_INSTANCE_ID)
```

Comment:
I think we want to use DiscardAttributeChecker.UNDEFINED here (and for other version transformations), since value of true should result in rejection.

Reply:
I assumed that those checkers were applied in sequence, i.e., fist if attribute is defined, it would be rejected. If not, apply next on the list: discard always, which would imply discard undefined. I'm updating the PR with DiscardAttributeChecker.UNDEFINED

Reply:
This broke transformation tests, I see errors such as the one below:
`[ERROR]   UndertowTransformersTestCase.testTransformersEAP_7_1_0:72->testTransformers:284->testTransformers:302->AbstractSubsystemTest.checkSubsystemModelTransformation:206->AbstractSubsystemTest.checkSubsystemModelTransformation:220 : {
 expected:<...-host, instance-id, []server, servlet-cont...> but was:<...-host, instance-id, [obfuscate-instance-id, ]server, servlet-cont...>`

For that reason, I'm reverting this change, @pferraro